### DISCUSSION TITLE
Alternative for Zone.CreateFromUrl in System.Security.Policy

### DISCUSSION THREAD

**Discussion by chaosbolt** [dotnet/runtime#89344] [General]
Is the concept of Zones defined in IE user settings still valid in dotnet core ?
If yes, then which API can be used to get the Zone of a given URL ?
In .net framework versions earlier, Zone.CreateFromUrl in System.Security.Policy was being used for the same.
Can anybody please suggest any alternative for this in dotnet core ?

**Comment by teo-tsirpanis** [comment]
Any form of sandboxing and code security is unavailable on modern .NET. You should use operating system facilities such as containers to run untrusted code. [^1]

[^1]: https://learn.microsoft.com/en-us/dotnet/core/porting/net-framework-tech-unavailable#code-access-security-cas
